src/main/studio/ui/chart/Chart.java

```diff
@@ -43,6 +47,8 @@ public class Chart implements ComponentListener {
     private static final Logger log = LogManager.getLogger();
     private static final Config config = Config.getInstance();
 
+    public final static int menuShortcutKeyMask = java.awt.Toolkit.getDefaultToolkit().getMenuShortcutKeyMask();
```

Comment:
`getMenuShortcutKeyMask` is depreated since java 10 - new method should be `getMenuShortcutKeyMaskEx`

Reply:
We can't use that in Java 8.